questionary/prompts/text.py

```diff
@@ -18,6 +18,8 @@ def text(
     validate: Any = None,
     qmark: Text = DEFAULT_QUESTION_PREFIX,
     style: Optional[Style] = None,
+    multiline: bool = False,
```

Comment:
should this rather be its own type instead of a flag? e.g. `textarea` 

@kiancross what do you think?

Reply:
@tmbo I think as types are defined by functions, and there is significant overlap between these two types, using a flag is better.

In an ideal world, I think the types would be implemented as classes, then someone like this could be done with minimal code repetition using inheritance.

Reply:
@tmbo Can you think of any other ways a multi-line input may diverge from a text input? If not, I think using the same type is ok.

Reply:
It uses a few more attributes as you can read in upstream docs, but personally I think it is ok as an attribute.

Reply:
Will leave this unresolved for @tmbo to comment.

Reply:
sounds good to me, let's keep it an attribute then